### PR TITLE
TCES-21: Activity timeline

### PR DESCRIPTION
**Pull request by thardchi**
# Description

Link to ticket: https://github.com/uoftblueprint/tces/issues/21

I implemented MUI's timeline component. I also installed MUI and MUI lab. Here are the list of new dependencies I added:
    "@emotion/react": "^11.10.5",
    "@emotion/styled": "^11.10.5",
    "@mui/lab": "^5.0.0-alpha.108",
    "@mui/material": "^5.10.14",

Here is a screenshot of what the component looks like:
Screen Shot 2022-11-19 at 10 17 37 PM

Fixes # 21

## Sample curl
Ignore if PR does not affect an endpoint.
## Response
Ignore if PR does not affect an endpoint.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] I have run `npm run lint:fix`
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (Delete if frontend)

### CODE REVIEW

**Review by EclipseIsDead — APPROVED** [review]
lgtm

File: client/src/App.tsx [hunk 1]
```diff
@@ -2,6 +2,42 @@ import React, { useEffect, useState } from 'react';
...
+import TimelineItem from '@mui/lab/TimelineItem';
+import TimelineSeparator from '@mui/lab/TimelineSeparator';
+import TimelineConnector from '@mui/lab/TimelineConnector';
+import TimelineContent from '@mui/lab/TimelineContent';
+import TimelineDot from '@mui/lab/TimelineDot';
+
+//implementing activityTimeline function
+export default function BasicTimeline() {
+  return (
+    <Timeline>
```
Comment (annie951977) [inline client/src/App.tsx:16]: maybe a comment describing how the components nest would be nice

**Review by annie951977 — APPROVED** [review]
lgtm, this is more of a suggestion but adding a comment to describe how the components are nested together would be nice for readability

**Comment by edwardhuahan** [comment]
Hey Thardchi would it be possible to move everything into it's own separate page/component? It would be great for just like splitting things up. Lmk if you need help with that or if anything was unclear

**Pull request closed without merging**